src/applications/mhv-medical-records/components/CareSummaries/ProgressNoteDetails.jsx

```diff
@@ -115,12 +116,9 @@ ${record.note}`;
       />
       <DownloadingRecordsInfo allowTxtDownloads={allowTxtDownloads} />
 
-      <div className="test-details-container max-80" data-dd-privacy="mask">
+      <div className="test-details-container max-80">
         <h2>Details</h2>
-        <h3
-          className="vads-u-font-size--base vads-u-font-family--sans"
-          data-dd-privacy="mask"
-        >
+        <h3 className="vads-u-font-size--base vads-u-font-family--sans">
```

Comment:
Thanks for removing the mask from this `<h3>`. You also need to remove it from the other `<h3>`s in this section.